microsoft-edge/index.yml

```diff
@@ -0,0 +1,445 @@
+### YamlMime:Hub
+
+title: Microsoft Edge developer documentation
```

Comment:
Do we need to say "developer" here? Our Enterprise content is more geared to IT pros and I'm sure we'll publish docs for other Edge customers as well. Can we change this to "Microsoft Edge documentation"

Reply:
@zoherghadyali coming here to +1 this. This should just be Microsoft Edge documentation.